source/common/http/http2/conn_pool.h

```diff
@@ -76,6 +79,8 @@ class ConnPoolImpl : public ConnectionPool::Instance, public ConnPoolImplBase {
 
   // Http::ConnPoolImplBase
   void checkForDrained() override;
+  void checkForIdle() override { /* TODO(klarose): implement */
+  }
```

Comment:
nit: NOT_IMPLEMENTED_GCOVR_EXCL_LINE

Reply:
This is actually being invoked by the base class when a pending request is cancelled. I had this in originally, but it failed because this line panicked in a UT.